src/layer/AbstractSentinelHubV3WithCCLayer.ts

```diff
@@ -0,0 +1,49 @@
+import { BBox } from 'src/bbox';
+import { PaginatedTiles } from 'src/layer/const';
+import { AbstractSentinelHubV3Layer } from './AbstractSentinelHubV3Layer';
+
+// this class provides any SHv3-specific functionality to the subclasses:
```

Comment:
Maybe change the comment a little so it's not exactly the same as in `src/layer/AbstractSentinelHubV3Layer.ts`? Maybe in the same fashion as in `AbstractSentinelHubV1OrV2WithCCLayer.ts`.
For example: `// same as AbstractSentinelHubV3Layer, but with maxCloudCoverPercent (useful for Landsat datasets)`

Reply:
Ok, I changed it to `// same as AbstractSentinelHubV3Layer, but with maxCloudCoverPercent (for layers which support it)`.

Resolving